data/mods/Aftershock/recipes/uncraft.json

```diff
@@ -107,11 +107,335 @@
+    "components": [
+      [ [ "ai_module", 1 ] ],
+      [ [ "robot_controls", 1 ] ],
+      [ [ "sensor_module", 1 ] ],
+      [ [ "spidery_legs_small", 1 ] ],
+      [ [ "afs_circuitry_1", 4 ] ],
+      [ [ "afs_circuitry_2", 2 ] ],
+      [ [ "afs_circuitry_3", 1 ] ],
+      [ [ "afs_energy_storage_3", 1 ] ],
+      [ [ "afs_material_1", 4 ] ],
+      [ [ "afs_material_2", 2 ] ],
+      [ [ "targeting_module", 1 ] ],
+      [ [ "pathfinding_module", 1 ] ]
+    ]
+  },
+  {
+    "result": "broken_defbot_shot",
+    "type": "uncraft",
+    "activity_level": "LIGHT_EXERCISE",
+    "time": "2 h",
+    "skill_used": "electronics",
+    "difficulty": 4,
+    "using": [ [ "soldering_standard", 20 ], [ "welding_standard", 5 ] ],
+    "qualities": [ { "id": "SCREW_FINE", "level": 1 }, { "id": "SAW_M", "level": 1 } ],
+    "components": [
+      [ [ "ai_module", 1 ] ],
+      [ [ "robot_controls", 1 ] ],
+      [ [ "sensor_module", 1 ] ],
+      [ [ "defbot_shot_chassis", 1 ] ],
+      [ [ "afs_circuitry_1", 6 ] ],
+      [ [ "afs_circuitry_2", 4 ] ],
+      [ [ "afs_circuitry_3", 2 ] ],
+      [ [ "afs_energy_storage_3", 2 ] ],
+      [ [ "afs_energy_storage_4", 1 ] ],
+      [ [ "afs_material_1", 12 ] ],
+      [ [ "afs_material_2", 4 ] ],
+      [ [ "targeting_module", 1 ] ],
+      [ [ "gun_module", 1 ] ]
+    ]
+  },
+  {
+    "result": "broken_milbot_556",
+    "type": "uncraft",
+    "activity_level": "LIGHT_EXERCISE",
+    "time": "2 h",
+    "skill_used": "electronics",
+    "difficulty": 5,
+    "using": [ [ "soldering_standard", 20 ], [ "welding_standard", 5 ] ],
+    "qualities": [ { "id": "SCREW_FINE", "level": 1 }, { "id": "SAW_M", "level": 1 } ],
+    "components": [
+      [ [ "ai_module", 1 ] ],
+      [ [ "robot_controls", 1 ] ],
+      [ [ "sensor_module", 1 ] ],
+      [ [ "milbot_humaniform_chassis", 1 ] ],
+      [ [ "afs_circuitry_1", 6 ] ],
+      [ [ "afs_circuitry_2", 4 ] ],
+      [ [ "afs_circuitry_3", 2 ] ],
+      [ [ "afs_energy_storage_3", 2 ] ],
+      [ [ "afs_energy_storage_4", 1 ] ],
+      [ [ "afs_material_1", 12 ] ],
+      [ [ "afs_material_2", 4 ] ],
+      [ [ "targeting_module", 1 ] ],
+      [ [ "gun_module", 1 ] ],
+      [ [ "pathfinding_module", 1 ] ],
+      [ [ "identification_module", 1 ] ]
+    ]
+  },
+  {
+    "result": "broken_milbot_needle",
+    "type": "uncraft",
+    "activity_level": "LIGHT_EXERCISE",
+    "time": "2 h",
+    "skill_used": "electronics",
+    "difficulty": 5,
+    "using": [ [ "soldering_standard", 20 ], [ "welding_standard", 5 ] ],
+    "qualities": [ { "id": "SCREW_FINE", "level": 1 }, { "id": "SAW_M", "level": 1 } ],
+    "components": [
+      [ [ "ai_module", 1 ] ],
+      [ [ "robot_controls", 1 ] ],
+      [ [ "sensor_module", 1 ] ],
+      [ [ "milbot_humaniform_chassis", 1 ] ],
+      [ [ "afs_circuitry_1", 6 ] ],
+      [ [ "afs_circuitry_2", 4 ] ],
+      [ [ "afs_circuitry_3", 2 ] ],
```

Comment:
This is another one that should probably have more of the advanced circuitry.

Reply:
If the recipe requires it to give more in the future, I'm willing to, but considering how common military robots will be, I'd rather I'd stay at tier 3 max

Reply:
I was more thinking afs_circuitry_1 9, afs_circuitry_2 6, afs_circuitry_3 3. (There's at least one other with afs_circuitry_1 3, afs_circuitry_2 2, afs_circuitry_3 1, so I'm extrapolating proportions.)